<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestSetup</class>
 <widget class="QDialog" name="TestSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2055</width>
    <height>904</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>30</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>时序：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cBox_panel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_testset_set">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>40</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_testset_load">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>40</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>加载配置文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_testset_test">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>40</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>测试</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_ChildImage">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>40</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>记录子图像</string>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_AutoFocus">
   <property name="geometry">
    <rect>
     <x>1480</x>
     <y>40</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>打开自动对焦</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_testset_2_Q">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>识别二维码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_messageTest">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>250</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>message</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_light">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开灯</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_handle_image">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>识别前处理图像</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_cut_image">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>140</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>剪切图像</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>130</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>二值化阀值：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_binF">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>140</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1370</x>
     <y>140</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>曝光时间：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_pox">
   <property name="geometry">
    <rect>
     <x>1560</x>
     <y>130</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>3000</number>
   </property>
   <property name="singleStep">
    <number>20</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_qrCode">
   <property name="geometry">
    <rect>
     <x>1340</x>
     <y>230</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>打开二维码识别</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_machine_back">
   <property name="geometry">
    <rect>
     <x>1300</x>
     <y>700</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/back.png);</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
